### PR TITLE
V3.1 changes

### PR DESCRIPTION
**Pull request by nigelpain**
These are changes to v3.1 requested from UR guys. Mainly amendments to wording of help text on the borrower flow, removing the third column from the summary screen table and adding an additional provisions page in at step 7.

### CODE REVIEW

File: app/views/v3-1/deed-journeys/deed-transaction/obligations.html [hunk 1]
```diff
@@ -14,30 +14,40 @@
...
-					<span class="heading-secondary">Step 6 of 6</span>
+					<span class="heading-secondary">Step 6 of 7</span>
 					Obligation
 				</h1>
 
 				<div class="text division-primary">
 					<p class="lede">
-						The Society has power under the mortgage conditions to make further
+						Bank of England PLC has power under the mortgage conditions to make further
 						advances secured by the mortgage and is obliged to do so.
```
Comment (danielmarker) [inline app/views/v3-1/deed-journeys/deed-transaction/obligations.html:24]: is or are here as its a company?

File: app/views/v3-1/deed-journeys/deed-transaction/summary.html [hunk 2]
```diff
@@ -101,20 +88,32 @@ <h1 class="heading-large">Review your mortgage deed</h1>
...
-						</td>
 					</tr>
 					<tr>
 						<th scope="row">Obligation</th>
 						<td>
 							<div class="text">
-								The Society has power under the mortgage conditions to make further
+								Bank of England PLC has power under the mortgage conditions to make further
 								advances secured by the mortgage and is obliged to do so.
 							</div>
```
Comment (danielmarker) [inline app/views/v3-1/deed-journeys/deed-transaction/summary.html:98]: Same comment on is and are

**Comment by danielmarker** [comment]
Couple comments on english reading - other wise technically all ok :+1:

**Comment by Phemus** [comment]
:+1: from me. we don't have control over the charging clause wording as its legal.

**Comment by danielmarker** [comment]
Happy to Approve then

On 17 September 2015 at 11:51, Ramin Vazir notifications@github.com wrote:

> [image: :+1:] from me. we don't have control over the charging clause
> wording as its legal.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/LandRegistry/charges-html-prototypes/pull/131#issuecomment-141041784
> .

## 

_Dan Marker_
_Digital Charges Team - Land Registry_
_Information Systems_

**Pull request merged**